deploy/kne/external-multinode.yaml

```diff
@@ -28,3 +28,6 @@ controllers:
   - kind: Lemming
     spec:
       operator: ../../manifests/controllers/lemming/manifest.yaml
+  - kind: Cdnos
```

Comment:
please remove from these examples, instead feel free to create a new example (kind-bridge-cdnos.yaml or similar)

Reply:
removed from examples and created new examples:
`kind-bridge-cdnos.yaml`
`external-multinode-cdnos.yaml`